src/main/webapp/app/overview/exercise-details/request-feedback-button/request-feedback-button.component.ts

```diff
@@ -62,6 +70,11 @@ export class RequestFeedbackButtonComponent implements OnInit {
             this.exerciseService.getExerciseDetails(this.exercise().id!).subscribe({
                 next: (exerciseResponse: HttpResponse<ExerciseDetailsType>) => {
                     this.participation = this.participationService.getSpecificStudentParticipation(exerciseResponse.body!.exercise.studentParticipations ?? [], false);
+                    if (this.participation) {
+                        this.currentFeedbackRequestCount =
+                            this.participation.results?.filter((result) => result.assessmentType == AssessmentType.AUTOMATIC_ATHENA && result.successful == true).length ?? 0;
+                        this.subscribeToResultUpdates();
```

Comment:
I would move that call to the ngoninit method as it is not really related to this

Reply:
I think I need to make this call here since I am waiting for participation data, otherwise, `subscribeToResultUpdates` would just return, and I can't display the current number of requests dynamically